controllers/package_controller_test.go

```diff
@@ -165,6 +170,89 @@ func TestReconcile(t *testing.T) {
 			t.Errorf("expected <%s> got <%s>", expected, got.RequeueAfter)
 		}
 	})
+
+	t.Run("Reports error when requested package version is not in the bundle", func(t *testing.T) {
+		tf, ctx := newTestFixtures(t)
+
+		fn, pkg := tf.mockGetFnPkg()
+		tf.ctrlClient.EXPECT().
+			Get(ctx, gomock.Any(), gomock.AssignableToTypeOf(pkg)).
+			DoAndReturn(fn)
+
+		tf.bundleManager.FakeActiveBundle = tf.mockBundle()
+
+		testErr := errors.New("status update test error")
+		status := tf.mockStatusWriter()
+
+		pkg.Spec.PackageVersion = "2.0.0"
+		pkg.Status.TargetVersion = "2.0.0"
+		pkg.Status.Detail = fmt.Sprintf("Package %s@%s is not in the current active bundle. Did you forget to activate the new bundle?", pkg.Spec.PackageName, pkg.Spec.PackageVersion)
```

Comment:
```suggestion
		pkg.Status.Detail = fmt.Sprintf("Package %s@%s is not in the active bundle (). Did you forget to activate the new bundle?", pkg.Spec.PackageName, pkg.Spec.PackageVersion)
```
Heh, there's no bundle version in `tf.mockBundle()`